### ISSUE TITLE
display the rva of the stack address and the unnamed address in the thread stack window

### ISSUE THREAD

**Issue by romanholidaypancakes** [winsiderss/systeminformer#1635]
### Description of the feature, modification, idea or suggestion

Currently I have to manually calculate rva(relative to the module (memory page) base address)  and address

![image](https://user-images.githubusercontent.com/51652878/223960158-0c875085-03a5-46d5-b57f-648199de9fb4.png)

**Comment by dmex** [comment]
Right-click the columns and add `Control address` which is the address of the symbol:

![image](https://user-images.githubusercontent.com/1306177/224046300-f1f2678b-c516-4390-bf19-274bc036c3ba.png)

 > calculate rva

The address is shown but not the RVA
